cmd/mesh/manifest-apply.go

```diff
@@ -39,6 +45,8 @@ type manifestApplyArgs struct {
 func addManifestApplyFlags(cmd *cobra.Command, args *manifestApplyArgs) {
 	cmd.PersistentFlags().StringVarP(&args.inFilename, "filename", "f", "", filenameFlagHelpStr)
 	cmd.PersistentFlags().StringVarP(&args.kubeConfigPath, "kubeconfig", "c", "", "Path to kube config.")
+	cmd.PersistentFlags().StringVar(&args.context, "context", "", "The name of the kubeconfig context to use.")
+	cmd.PersistentFlags().StringVarP(&args.namespace, "namespace", "n", v1.NamespaceAll, "Namespace for the kubeConfig.")
```

Comment:
Is this really required? IstioControlPlane has defaultNamespace which defaults to istio-system in the profiles.

Reply:
Removed `namespace` flag.